docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -1,164 +1,145 @@
+
+        The Properties window is displayed. Use the **GUID** value.
+
+    For more information about DS8000 Family products, find your product information in [IBM Documentation](https://www.ibm.com/docs/).
+  
+
+Use this procedure to help build a PV YAML file for your volumes.
+
+**Note:** These steps are set up for importing volumes from a Spectrum Virtualize Family system. Change parameters, as needed.
+
+1. Create a persistent volume (PV) YAML file.
+
+    **Important:** Be sure to include the `storageClassName` and `controllerPublishSecretRef` parameters or errors may occur.
 
-3.  Create a PersistentVolumeClaim (PVC) yaml file.
+2. Take the `volume_name` and other optional information (collected before the procedure) and insert it into the YAML file (under `spec.csi.volumeAttributes`).
+
+    **Important:** If using the CSI Topology feature, the `spec.csi.volumeHandle` contains the management ID (see [Creating a StorageClass with topology awareness](csi_ug_config_create_storageclasses_topology.md)). In the example below, the `spec.csi.volumeHandle` would read similar to the following: `SVC:demo-system-id-1:0;600507640082000B08000000000004FF`.
+    
+        apiVersion: v1
+        kind: PersistentVolume
+        metadata:
+          # annotations:
+            # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
+          name: demo-pv
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          capacity:
+            storage: 1Gi
+          csi:
+            controllerPublishSecretRef:
+              name: demo-secret
+              namespace: default
+            driver: block.csi.ibm.com
+            # volumeAttributes:
+              # pool_name: demo-pool
+              # storage_type: SVC
+              # volume_name: demo-prefix_demo-pvc-file-system
+              # array_address: demo-management-address
+            volumeHandle: SVC:0;600507640082000B08000000000004FF
+          # persistentVolumeReclaimPolicy: Retain
+          storageClassName: demo-storageclass
+          # volumeMode: Filesystem
+
+3. Create a PersistentVolumeClaim (PVC) YAML file.
 
     **Note:**
 
-    -   To include a specific 5 Gi PV, be sure to include the `storageClassName`.
-    -   For more information about creating a PVC yaml file, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+    - Be sure to include the `storageClassName`.
+    - For more information about creating a PVC YAML file, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
     
-    ```screen
+    ```
     apiVersion: v1
     kind: PersistentVolumeClaim
     metadata:
-      # annotations:
-        # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
-      name: vol1-pvc
+      name: demo-pvc
     spec:
       accessModes:
       - ReadWriteOnce
       resources:
         requests:
           storage: 1Gi
       storageClassName: ibmc-block-gold
```

Comment:
as was [done for the pv](https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r713186517):
`ibmc-block-gold` -> `demo-storageclass`

Reply:
I will add this to the PR for 1.9.0

Reply:
thanks, fixed in https://github.com/IBM/ibm-block-csi-driver/pull/466